tailwind.config.ts

```diff
@@ -19,10 +19,10 @@ const config: Config = {
     },
   },
   daisyui: {
-    darkTheme: "customDark",
+    darkTheme: "dark",
     themes: [
       {
-        customLight: {
+        light: {
```

Comment:
Reverting to default names helped me avoid passing around the custom names in my components. Everything seems to work as expected but I'm not sure whether I'm unknowingly breaking something here.

Reply:
Don't believe so - when I was playing around with this, I had the impression that Tailwind uses the first theme by default. I like this! ✅